src/app/app.component.html

```diff
@@ -1 +1,29 @@
-<router-outlet></router-outlet>
+<head>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+<div class="container">
+    <div class="header">
+        <div class="logo">
+            <mat-icon id="logo">pets</mat-icon>
+        </div>
+        <div class="navbar">
+            <nav>
+                <a routerLink="/home" routerLinkActive="active">Home</a>
+                <a routerLink="/about" routerLinkActive="active">About</a>
+                <a routerLink="/bookings" routerLinkActive="active">Bookings</a>
```

Comment:
We store these as Appointments on the BE, maybe it would be better to sync on this change so the router link matches the ednpoint. @MuratUseini123 is working on the appointments model and endpoint, have you already configured this in some way?

Reply:
![image](https://user-images.githubusercontent.com/73704530/235364674-21e0a0b9-9f5b-4803-b2d3-eab76906b3b6.png)

It is changed to "Appointments" now.